ingest-app/src/migrations/009_update_bulk_status_message.clj

```diff
@@ -0,0 +1,17 @@
+(ns migrations.009-update-bulk-status-message
+  (:require
+   [clojure.java.jdbc :as j]
+   [config.migrate-config :as config]))
+
+(defn up
+  "Migrates the database up to version 9."
+  []
+  (println "migrations.009-update-bulk-status-message up...")
+  (j/db-do-commands (config/db)
+                    "ALTER TABLE bulk_update_coll_status MODIFY
+                     (status_message VARCHAR(4000))"))
+
+(defn down
```

Comment:
Won't you need to truncate here? I'm imagining running migrations locally (up and down) when large status messages have been inserted into the DB ... I think there'd be a migration failure in that case?